pkg/security/apparmor/validate.go

```diff
@@ -133,7 +133,7 @@ func getAppArmorFS() (string, error) {
 			continue
 		}
 		if fields[2] == "securityfs" {
-			appArmorFS := path.Join(fields[1], "apparmor")
+			appArmorFS := filepath.Join(fields[1], "apparmor")
```

Comment:
I was going to check if the returned path is used in ways that would make different slashes unsafe, but weirdly, it doesn't look like the returned path is used... cc @tallclair for general apparmor oversight

Reply:
apparmor is only used on linux, so this doesn't really matter here. But this is a file path, so this change here seems appropriate.

@liggitt I didn't understand your comment. This path `appArmorFS` is used in the following line to check whether the path exists.

Reply:
Ah, got it. Yeah, looks like the usage was removed in https://github.com/kubernetes/kubernetes/pull/97966.

This is still a proxy for whether AppArmor is enabled on the cluster, but we're checking `apparmor.IsEnabled()` directly, so I think the we can just remove this function entirely. :+1:

Reply:
Filed "help wanted" issue: https://github.com/kubernetes/kubernetes/issues/115741